HLTriggerOffline/SUSYBSM/python/SUSYBSM_HLT_Electron_BJet_cff.py

```diff
@@ -10,7 +10,7 @@
   TriggerPath = cms.string('HLT_Ele10_CaloIdM_TrackIdM_CentralPFJet30_BTagCSV_p13_v'),
   TriggerFilterEle = cms.InputTag('hltSingleEle10CaloIdTrackIdVLDphiFilter', '', 'HLT'), #the last filter in the path hltSingleEle10CaloIdTrackIdVLOneOEMinusOneOPFilterRegional
   TriggerFilterJet = cms.InputTag('hltCSVFilterSingleEle10', '', 'HLT'), #the last filter in the path
-  PtThrJet = cms.untracked.double(40.0),
+  PtThrJet = cms.untracked.double(30.0),

```

Comment:
Hi David,

While that might be simpler, the issue is that not all SUSY DQM modules use the same cut because of different jet requirements in the HLT and in the analyses.  Having a central constant would cause confusion in my opinion if it's not used everywhere.  So I'd prefer to keep this as is.

Thanks,
Dominick